# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../etoile/usr-local.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- Etoile-0.2/setup.sh.vanilla	2007-11-19 21:43:30.000000000 +0100
+++ Etoile-0.2/setup.sh	2007-11-19 21:43:54.000000000 +0100
@@ -27,7 +27,7 @@
 systemLevelSetup()
 {
 
-	tooldir="/usr/local/bin"
+	tooldir="/usr/X11/bin"
 	
 	#
 	# Install essential GNUstep tools in a convenient way
@@ -257,7 +257,7 @@
 
 if [ $AS_ROOT = yes -o $HAVE_SUDO = yes ]; then
     setupdir="$GNUSTEP_SYSTEM_ROOT/Library"
-    tooldir="/usr/local/bin"
+    tooldir="/usr/X11/bin"
 else
     setupdir="$GNUSTEP_USER_ROOT/Library"
     tooldir="$HOME/bin"
